dask/dataframe/tests/test_multi.py

```diff
@@ -2009,6 +2009,7 @@ def test_concat_datetimeindex():
     assert_eq(result, expected)
 
 
+@pytest.mark.skipif(PANDAS_GT_140, reason="Append is deprecated in pandas 1.4.0")
```

Comment:
Instead of skipping for `pandas >= 1.4.0` can we still run these tests but make sure the appropriate deprecation warning is emitted?

Reply:
Sure let me do that.